packages/clarity-visualize/src/heatmap.ts

```diff
@@ -72,8 +142,8 @@ export function click(activity: Activity): void {
 
 function overlay(): HTMLCanvasElement {
     // Create canvas for visualizing heatmap
-    let doc = state.player.contentDocument;
-    let win = state.player.contentWindow;
+    let doc = state.window.document;
+    let win = state.window;
     let de = doc.documentElement;
     let canvas = doc.getElementById(Constant.HeatmapCanvas) as HTMLCanvasElement;
     if (canvas === null) {
```

Comment:
should we check for options.canvas here?

Reply:
yes good point, will do

Reply:
thought about it more and left it as it is... it's hard to accomplish anything without canvas support in heatmap. will think this over and include it as separate change later.